source/common/ur_util.hpp

```diff
@@ -268,7 +268,7 @@ inline std::optional<EnvVarMap> getenv_to_map(const char *env_var_name,
             }
             values_vec.push_back(value);
         }
-        map[key] = values_vec;
+        map[key] = std::move(values_vec);
```

Comment:
I wonder whether we shouldn't just ignore issues like these. Adding extraneous `std::move` in non-performance critical code just adds clutter.